app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -333,6 +324,7 @@ protected void onPostExecute(String lyrics) {
                     }
                 } else {
                     lyricsInfo = new LyricsDialog.LyricInfo(song.title, lyrics);
+                    playerAlbumCoverFragment.setSynchronizedLyrics(AbsSynchronizedLyrics.parse(lyrics));
```

Comment:
Yeah, that was the general idea I had. I'm not familiar with the specifics of each, so I'll leave it up to your discretion on whether it's a good idea or not.